bettermod/bettermod.py

```diff
@@ -1757,16 +1758,17 @@ def clear_cache(self):
     @commands.group(pass_context=True, no_pm=True)
     @checks.mod_or_permissions(administrator=True)
     async def case(self, ctx):
-        """Edit warnings' reasons or remove them"""
+        """Edit or remove warning's reasons."""
```

Comment:
That's not clear; you edit a reason or remove a warning, but you don't remove a warning's reason.